shell/tests/apply_and_verify_context_actions_from_file.rs

```diff
@@ -134,16 +134,20 @@ fn feed_tezedge_context_with_actions() -> Result<(), Error> {
     let header_stub = BlockHeaderWithHash::new(block_header_stub).unwrap();
 
     let cache = Cache::new_lru_cache(128 * 1024 * 1024).unwrap(); // 128 MB
-    let commit_log_db_path = PathBuf::from("/tmp/commit_log/");
-    let key_value_db_path = PathBuf::from("/tmp/key_value_store/");
 
+    let out_dir_env = env::var("OUT_DIR")
```

Comment:
@mat13mn 
this is unrelated, but could you please all merkle_storage test change to create dirs to `env::var("OUT_DIR")` ?

Reply:
sure - i will check that

Reply:
just run the tests and everything seems file - that has been addressed in GC PR i guess